modules/widgets/customizer-utils.js

```diff
@@ -59,6 +59,12 @@ wp.isJetpackWidgetPlaced = function( placement, widgetName ) {
 					} else if ( wp.isJetpackWidgetPlaced( placement, 'eu_cookie_law_widget' ) ) {
 						// Refresh EU Cookie Law
 						placement.container.fadeIn();
+					} else if ( wp.isJetpackWidgetPlaced( placement, 'jetpack_simple_payments_widget' ) ) {
```

Comment:
For folks reading, internally this calls. So I think theoretically we should only have one of these fire at a time, based on what we're editing.
```
/**
 * Verifies that the placed widget ID contains the widget name.
 * @param {object} placement
 * @param {string} widgetName
 * @returns {*|boolean}
 */
wp.isJetpackWidgetPlaced = function( placement, widgetName ) {
	return placement.partial.widgetId && 0 === placement.partial.widgetId.indexOf( widgetName );
};
```